### PR TITLE
storage_benchmark: support compile iozone on aarch64

### PR DESCRIPTION
**Pull request by yanan-fu**
id: 1784810 

Signed-off-by: Yanan Fu <yfu@redhat.com>

### CODE REVIEW

**Comment by yanan-fu** [comment]
(1/1) Host_RHEL.m8.u2.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.aarch64.io-github-autotest-qemu.virtio_storage_in_use.with_vioscsi.data_disk.with_live_migration.before_bg_test.arm64-pci: PASS (240.36 s)

File: provider/storage_benchmark.py [hunk 1]
```diff
@@ -281,7 +281,9 @@ def __init__(self, params, session):
         host_path = os.path.join(data_dir.get_deps_dir(), 'iozone', iozone_pkg)
         self.download_path = os.path.join('/home', iozone_pkg)
         self.iozone_inst = os.path.join('/home', 'iozone_inst')
-        self.arch = 'linux-AMD64' if 'x86_64' in machine() else 'linux-powerpc64'
+        self.arch = 'linux-AMD64' if 'x86_64' in machine() \
```
Comment (xutian) [inline provider/storage_benchmark.py]: please split to multi-lines. use if else branch instead.

Comment (yanan-fu) [reply provider/storage_benchmark.py]: @xutian could you explain more ?  what's the defect of the current code since both of them can work.

**Comment by yanan-fu** [comment]
@PaulYuuu could you help review ? thanks!

**Comment by xutian** [comment]
LGTM

**Comment by PaulYuuu** [comment]
Ack.

**Comment by yanan-fu** [comment]
Thanks all!

**Pull request merged**